Resources/Prototypes/_CP14/Entities/Objects/Materials/ores.yml

```
@@ -184,3 +184,60 @@
   - type: Stack
     count: 10
 
+- type: entity
+  id: CP14OreCoal1
+  parent: BaseItem
+  name: coal
+  description: delicious and burns well.
```

```suggestion
  description: Delicious and burns well.
```

also, why "delicious"?

---

Можно, но придется редачить оффовские системы чтобы подружить StackComponent и FoodComponent

---

что я бы предпочел не делать

---

Мммм, прикольно, я тоже этого не хочу делать... ну увы...